lang/es/lang.php

```diff
@@ -8,31 +8,45 @@
         'access_users' => 'Administrar usuarios',
         'access_groups' => 'Administrar grupos',
         'access_settings' => 'Preferencias de usuario',
+        'impersonate_user' => 'Personificar Usuario'
     ],
     'users' => [
         'menu_label' => 'Usuarios',
         'all_users' => 'Todos los usuarios',
         'new_user' => 'Nuevo usuario',
         'list_title' => 'Administrar usuarios',
-        'activating' => 'Activando...',
```

Comment:
Why was this key removed?

Reply:
I couldn't find it in the english file (ran a diff). I did some digging and it was removed from EN in 7b47166bc7a4f47dba139d54f9d0d9622fc3ce41

Reply:
Gotcha, thanks for checking!